### ISSUE TITLE
"Choose catalog/schema" dialog works badly with keyboard

### ISSUE THREAD

**Issue by kikonen** [dbeaver/dbeaver#18907]
### Description

Selecting from DB in left side of "Choose catalog/schema" dialog is pretty much impossible using keyboard (while can get into dialog via keyboard ctrl+0), since every key press steals focus and moves it into right side of schema selection (which likely has only single "public" entry anyhow...)

Thus have to use mouse to for selection which makes dialog really akward.

![image](https://user-images.githubusercontent.com/478935/215513824-11d8b6a5-474b-48ee-a6ad-e99ed5e8bf84.png)

### DBeaver Version

Version 22.3.3.202301211053

### Operating System

Windows 11

### Database and driver

Postgres

### Steps to reproduce

1. CTRL + 0 to open "database/schema" selector
2. Try to select DB from left side, via keyboard navigation

=> fails, since focus is stolen to right side list after any key press

**Comment by Destrolaric** [comment]
@kikonen Hi, I see that any catalog selection in the first tree leads to the focus being set to schema selection. We should look at how it can be fixed.

**Comment by kikonen** [comment]
Biggest problem is that when left side panel is focused (well, it should be focused initially, don't change that), then every key press moves focus to right hand side (of course tabulating between those panes is also neither much fun, due to all extra controls in between in tabulation order; I guestion usefullness of those controls :)).

**Comment by kikonen** [comment]
Works still very badly.

Also the fact that first must select datasource, and only after that can select actual DB via separates dialog is clumsy. Especially since this other diallog works *only* after purely random delay, and selecting entries in dialog is *extremely* slow, after every key press app is pending very long time until responding into next key press.

Most of time in dialog goes in watching "connecting", after every key press.

![image](https://github.com/dbeaver/dbeaver/assets/478935/b40033f2-32b7-4bc0-8e6e-69d4708dd9d7)

So UX is very frustrating and slow.

**Comment by E1izabeth** [comment]
@kikonen please create a separate issue for that as current issue about focus was fixed and closed
